src/BIAnalyticsDataService/CreateParticipantScreeningEpisode/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS installer-env
+
+COPY ./AnalyticsDataService/SaveTransformedData /src/dotnet-function-app
```

Comment:
Shouldn't the path be `./BIAnalyticsDataService/CreateParticipantScreeningEpisode`?

Reply:
ah will correct